src/System.Security.Permissions/src/System/Security/Permissions/MediaPermission.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+namespace System.Security.Permissions
+{
+    public enum MediaPermissionAudio
+    {
+        NoAudio,
+        SiteOfOriginAudio,
+        SafeAudio,
+        AllAudio
+    }
+    public enum MediaPermissionVideo
+    {
+        NoVideo,
+        SiteOfOriginVideo,
+        SafeVideo,
+        AllVideo,
+    }
+    public enum MediaPermissionImage
+    {
+        NoImage,
+        SiteOfOriginImage,
+        SafeImage,
+        AllImage,
+    }
+    [Serializable()]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35")]
+    sealed public class MediaPermission : CodeAccessPermission, IUnrestrictedPermission
+    {
+        public MediaPermission() { }
+        public MediaPermission(PermissionState state) { }
+        public MediaPermission(MediaPermissionAudio permissionAudio) { }
+        public MediaPermission(MediaPermissionVideo permissionVideo) { }
+        public MediaPermission(MediaPermissionImage permissionImage) { }
+        public MediaPermission(MediaPermissionAudio permissionAudio,
+                               MediaPermissionVideo permissionVideo,
+                               MediaPermissionImage permissionImage)
+        { }
```

Comment:
Why wouldn't you continue throwing on bad input?

Reply:
Nevermind, looks like we really did just hollow everything out in this assembly. (I thought argument validation remained)